ecs-init/VERSION

```diff
@@ -0,0 +1 @@
+1.12.2
```

Comment:
Can you update this to `1.13.0` and run the `update-version.sh` script to generate a new version file?